calculator.rb

```diff
@@ -0,0 +1,62 @@
+def div_num(num1, num2)
+  unless num2 == 0
+    return num1 / num2
+  else
+    puts "You can't really divide numbers by 0... Awkward.."
+  end
+end 
+
+def check_num
```

Comment:
This is a minor suggestion, but: Instead of the name `check_num`, there may be a better name for this. What is this method responsible for? Since it's responsible for getting user input until it's valid, I may call it `get_valid_user_input`, but there are definitely a lot of other options that are good too.